libzhlgen/test/functions/win32/1.6.9/ResourceControl.zhl

```diff
@@ -14,8 +16,8 @@ __thiscall int ResourceControl::RenderImage(ResourceControl *this, GL_Texture* t
 __thiscall int ResourceControl::RenderImageString(ResourceControl *this, std::string& tex, int x, int y, int rotation, GL_Color color, float opacity, bool mirror);
 ".578d7c240883e4f0ff77fc5589e557565383ec5c8b570c8b07f30f1077208b5f048b7708f30f1175d08955d48b5724890424":
 __thiscall GL_Primitive* ResourceControl::CreateImagePrimitiveString(ResourceControl *this, const std::string& tex, int x, int y, int rotation, GL_Color color, float alpha, bool mirror);
-"578d7c240883e4f0ff77fc5589e557565389cb83ec4c8b37":
-__thiscall freetype::font_data &ResourceControl::GetFontData(ResourceControl *this, int fontType, bool unk);
+"57????????83e4f0ff77fc5589e557565389cb83ec4c8b37897dc08b4f048d46fa83f8020f87????????c70709000000b809000000ba01000000be09000000":
+__thiscall freetype::font_data &ResourceControl::GetFontData(ResourceControl *this, int size, bool ignoreLanguage);
```

Comment:
Make this same change in the ELF_x86 & ELF_amd64 signatures so that the argument names match